src/reducers/mergeReducer.js

```diff
@@ -22,19 +22,6 @@ const initialState = {
     disableInput: '',
     blah: ''
   },
-//     input: 'AGR:AGR-Reference-0000569189',
```

Comment:
These comments are helpful for telling people what to test when wanting to try different merge functionality, and are probably easier than keeping track of them in a separate document.

Reply:
Okay if you find them useful I am happy to add them back.

Reply:
Cool, thanks :)